purchase_representative/hooks.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tools import pycompat
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+def post_init_hook(cr, registry):
```

Comment:
Maybe a post_uninstall_hook is necessary, no?

Reply:
Done!